.github/workflows/SyncToAtlas.yml

```diff
@@ -0,0 +1,54 @@
+name: Sync Master to Atlas Repos
+
+on: 
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  atlas:
+
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@master 
+    - name: Add O'Reilly remotes
+      run: |
+        git remote add oreilly git@git.atlas.oreilly.com:lenucksi/learningpath-introduction.git
+    - uses: fregante/setup-git-token@v1
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Where are these secrets stored?

Reply:
@rrrutledge they're stored in the project itself. this particular one is a builtin - but the others can be found by navigating to Settings | Secrets on the project...

Reply:
Exactly what @StingRayZA says.

Reply:
Thanks.